app/controllers/vacancies_controller.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class VacanciesController < ApplicationController
+  before_action :require_login, except: %i[index show]
```

Comment:
its funny how i like the old way more, like: `except: [:index, :show]`

Reply:
I prefer using %i. I can omit the `:`. And this is the styleguide